modules/ROOT/pages/release-notes/release-notes55.adoc

```diff
@@ -142,26 +143,26 @@ Four new menu items have been added to the Table plugin to assist with moving an
 * `tablepastecolumnbefore`
 * `tablepastecolumnafter`
 
-For information on the Table plugin menu items, see: link:{baseurl}/plugins/opensource/table/#menuitems[Table plugin - Menu items].
+For information on the Table plugin menu items, see: xref:plugins/opensource/table.adoc#menuitems[Table plugin - Menu items].
 
 === New `table_column_resizing` option for controlling table column resizing
 
 A new `table_column_resizing` option sets whether a table or other columns are resized when a user resizes a table column.
 
-For information on the `table_column_resizing` option, see: link:{baseurl}/plugins/opensource/table/#table_column_resizing[Table plugin - `table_column_resizing`].
+For information on the `table_column_resizing` option, see: xref:plugins/opensource/table.adoc#table_column_resizing[Table plugin - `table_column_resizing`].
```

Comment:
This xref doesn't resolve as the `table_column_resizing` id does exist on the heading in `table_column_resizing.adoc`. e.g:
```
[[table_column_resizing]]
=== `table_column_resizing`
```

Reply:
done